src/CoreShop/Bundle/StorageListBundle/EventListener/CacheListener.php

```diff
@@ -44,41 +41,7 @@ public static function getSubscribedEvents(): array
 
     public function onKernelResponse(ResponseEvent $event): void
     {
-        if ($this->pimcoreContext->matchesPimcoreContext($event->getRequest(), PimcoreContextResolver::CONTEXT_ADMIN)) {
-            return;
-        }
-
-        if (!$event->isMainRequest()) {
-            return;
-        }
-
-        if ($event->getRequest()->attributes->get('_route') === '_wdt') {
-            return;
-        }
-
-        /** @var Request $request */
-        $request = $event->getRequest();
-
-        if (!$request->hasSession()) {
-            return;
-        }
-
-        try {
-            $list = $this->context->getStorageList();
-        } catch (StorageListNotFoundException) {
-            return;
-        }
-
-        if ($list->getId()) {
-            foreach ($list->getItems() as $item) {
-                if (!$item->getId()) {
-                    continue;
-                }
-
-                Cache::addIgnoredTagOnSave('object_' . $item->getId());
-            }
-
-            Cache::addIgnoredTagOnSave('object_' . $list->getId());
-        }
+        Cache::addIgnoredTagOnSave(sprintf('object_%s', $this->storageListRepository->getClassId()));
+        Cache::addIgnoredTagOnSave(sprintf('object_%s', $this->storageListItemRepository->getClassId()));
```

Comment:
```suggestion
        Cache::addIgnoredTagOnSave(sprintf('class_%s', $this->storageListRepository->getClassId()));
        Cache::addIgnoredTagOnSave(sprintf('class_%s', $this->storageListItemRepository->getClassId()));
```
Shouldn't it be `class` as prefix?